PULL_REQUEST_TEMPLATE.md

```diff
@@ -45,20 +41,14 @@
 
 > _Something brief that conveys the change and is written with the **persona (Alana, Cody...)** in mind. See [previous release notes](https://github.com/cloudfoundry/cf-deployment/releases) for examples._
 
-### Does this PR introduce a new BOSH release into the base cf-deployment.yml manifest or any ops-files?
-
-- [ ] YES - please specify
-- [ ] NO
-
-### Will this change increase the VM footprint of cf-deployment?
```

Comment:
We want to keep this section because there might be some non-obvious component changes that require an increase to VMs for it function properly. 

For Example, an extra addon or a new bosh job will require bigger diego cells.

Reply:
Interesting. Wouldn't that be a breaking change, because it adds a new job to CF (and thus increases vm footprint)?

Reply:
I also moved this to the breaking change section. See above